conda_build/source.py

```diff
@@ -59,6 +59,8 @@ def download_to_cache(metadata, config):
             try:
                 print("Downloading %s" % url)
                 download(url, path)
+            except CondaHTTPError as e:
+                print("Error: %s" % str(e).strip(), file=sys.stderr)
```

Comment:
would like these to be calls to logging rather than print, but I'm going to merge this as-is and refactor it next time I see it.